core/src/main/java/com/graphhopper/routing/util/BikeCommonFlagEncoder.java

```diff
@@ -248,8 +248,19 @@ public long acceptWay(ReaderWay way) {
         if (!highwaySpeeds.containsKey(highwayValue))
             return 0;
 
+        String sacScale = way.getTag("sac_scale");
```

Comment:
Nice, I can see why you moved this before the improved 'has bicylce tag' check. Would you mind to add a test for sac_scale that would fail if put afterwards?